lib/bosh/manifest/Header.js

```diff
@@ -14,8 +14,8 @@ class Header {
     this.disk_pools = header.disk_pools;
     this.resource_pools = header.resource_pools;
     this.networks = header.networks;
+    this.tags = header.tags;
```

Comment:
add this line only if tags is defined. This will take care of not having tags block when its undefined.
```
if(header.tags) {
this.tags = header.tags;
}
```